src/Validator.php

```diff
@@ -30,15 +30,6 @@ public function __construct(
         $this->translator = $translator;
         $this->translationDomain = $translationDomain;
         $this->translationLocale = $translationLocale;
-
-        foreach ($rules as $attribute => $ruleSets) {
```

Comment:
You should delete the `rules` parameter too

Reply:
done, thks.